client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -90,7 +90,7 @@ const CancelPrivateRegistration = React.createClass( {
 		this.props.cancelPrivateRegistration( id ).then( () => {
 			this.resetState();
 
-			notices.success( this.translate( 'You have successfully canceled private registration for %(domain)s.', {
+			notices.success( this.translate( 'You have successfully canceled privacy protection for %(domain)s.', {
```

Comment:
Warning: 20 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'You have successfully canceled the Private Registration upgrade for your domain.' )`&mdash; translations: **23**. *ES Score: 5.23*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/privacy-protection)